Instagram/ComposeViewController.m

```diff
@@ -18,44 +20,55 @@ - (void)viewDidLoad {
     self.caption.layer.borderWidth = 2.0f;
     self.caption.layer.borderColor = [[UIColor lightGrayColor] CGColor];
     self.caption.layer.cornerRadius = 5;
+}
 
+- (IBAction)onTap:(id)sender {
     UIImagePickerController *imagePickerVC = [UIImagePickerController new];
     imagePickerVC.delegate = self;
     imagePickerVC.allowsEditing = YES;
-
+    
```

Comment:
why? This makes me think it's copy pasta from codepath because I saw something similar before. 
Make sure you have this setting turned on in Xcode to avoid committing whitespace only changes.

![image](https://user-images.githubusercontent.com/4922820/125142038-8f2ce880-e0cb-11eb-8a71-d8ad790a431c.png)